vllm/v1/core/kv_cache_coordinator.py

```diff
@@ -166,6 +166,7 @@ def get_blocks(self, request_id: str) -> list[list[KVCacheBlock]]:
         return [
             manager.req_to_blocks[request_id]
             for manager in self.single_type_managers
+            if request_id in manager.req_to_blocks
```

Comment:
@NickLucche I think we'll want to actually return a list of empty lists in this case. I didn't update it yet since we might as well do it after rebasing on the other fix.
```suggestion
            manager.req_to_blocks.get(request_id) or []
            for manager in self.single_type_managers
```